### PR TITLE
Update setup-ha-etcd-with-kubeadm.md

### LINKED ISSUE #41420: migrate deprecated flags to config in kubeadm HA etcd page
When starting kubelet with the provided unit file `20-etcd-service-manager.conf` in https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm/, kubelet crashes, stating that all of the specified flags are either deprecated or completely unknown:
```
Jun 01 21:46:27 VF001695 kubelet[30937]: Flag --address has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 01 21:46:27 VF001695 kubelet[30937]: Flag --pod-manifest-path has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 01 21:46:27 VF001695 kubelet[30937]: Flag --cgroup-driver has been deprecated, This parameter should be set via the config file specified by the Kubelet's --config flag. See https://kubernetes.io/docs/tasks/administer-cluster/kubelet-config-file/ for more information.
Jun 01 21:46:27 VF001695 kubelet[30937]: E0601 21:46:27.359201   30937 run.go:74] "command failed" err="failed to parse kubelet flag: unknown flag: --container-runtime"
Jun 01 21:46:27 VF001695 systemd[1]: kubelet.service: Main process exited, code=exited, status=1/FAILURE
```
If this service file is no longer valid, the documentation should not include it.

### PR DESCRIPTION
**Pull request by tamilselvan1102**
fixes #41420

### CODE REVIEW

File: content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md [hunk 1]
```diff
@@ -63,12 +63,22 @@ on Kubernetes dual-stack support see [Dual-stack support with kubeadm](/docs/set
...
+   cat << EOF > /etc/systemd/system/kubelet.service.d/kubelet.conf
+   # Replace "systemd" with the cgroup driver of your container runtime. The default value in the kubelet is "cgroupfs".
+   # Replace the value of "containerRuntimeEndpoint" for a different container runtime if needed.
+   #
+   apiVersion: kubelet.config.k8s.io/v1beta1
+   kind: KubeletConfiguration
+   cgroupDriver: systemd
+   address: 127.0.0.1
+   containerRuntimeEndpoint: unix:///var/run/containerd/containerd.sock
+   podManifestPath: /etc/kubernetes/manifests
```
Comment (neolit123) [inline content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md:75]:
```suggestion
   staticPodPath: /etc/kubernetes/manifests
```

**Review by neolit123 — COMMENTED** [review]
sorry made mistake about one field
double check them
https://kubernetes.io/docs/reference/config-api/kubelet-config.v1beta1/

keep the commits squashed to one

File: content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md (hunk 1 above)
Comment (tamilselvan1102) [reply content/en/docs/setup/production-environment/tools/kubeadm/setup-ha-etcd-with-kubeadm.md]: @neolit123 
updated asper your suggestion

**Review by neolit123 — COMMENTED** [review]
/lgtm

**Comment by neolit123** [comment]
> issue: #41420

please change this to "fixes #41420"

**Comment by neolit123** [comment]
> fixes : #41420

should be without the :

**Comment by neolit123** [comment]
/assign @tengqm 
PTAL

**Pull request merged**